src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -14,6 +14,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_REMARK = new Prefix("r/");
     public static final Prefix PREFIX_DATE_OF_BIRTH = new Prefix("dob/");
     public static final Prefix PREFIX_INCOME = new Prefix("income/");
+    public static final Prefix PREFIX_FAMILY_SIZE = new Prefix("familysize/");
```

Comment:
Seems slightly long for prefix, maybe can consider shortening to "fs/".

Reply:
I think `fs` might be too short and not intuitive enough? Perhaps I'll shorten the prefix to `famsize` in milestone 1.5.